### ISSUE TITLE
Creating 3d bounding on Clusteres obtained

### ISSUE THREAD

**Issue by Sehjbir** [ajith3530/Python_EUCLIDEAN_clustering#2]
Great work on clustering the 3d points. Post clustering i want to create a bounding box on the clusters. Any method on how to do it ? 

Plus if i need to draw a plan or line through all the clustered points ? should i use Ransac to do so ?

**Comment by ajith3530** [comment]
I am not sure of any algo as such for creating bounding boxes, if you know the centroid, finding the width along each axis could give you a decent estimate of the bounding box location.
Ransac would be one of the ways to go ahead, basically you are looking at any of the curve fitting methods available.
Rasac is simpler to understand, but requires lots of tuning and optimizations to get it into a working shape.

**Comment by Sehjbir** [comment]
Yes i agree. I ran Ransac with iterations 50 and distance_thres = 1. However, i got an empty outlier DF. 
Currently running it with distance_thres = 0.01 and iteration = 50. 
I haven't voxel filtering on this. Should i do voxel filtering first and than run the ransac algo ? 
Post Ransac my aim is to do clustering of similar points and than draw a oriented box over the clusters in 3d space in python

**Comment by ajith3530** [comment]
Voxel filtering should be first, There is a formula available for calculating the number of iterations for ransac. 50 iterations seems to be rather too less, I don't remember it now, but I read it on researchgate
